lib/Doctrine/DBAL/Query/Expression/CompositeExpression.php

```diff
@@ -84,6 +88,28 @@ public function add($part)
         return $this;
     }
 
+    /**
+     * Returns a CompositeExpression with the given part added.
```

Comment:
Should it be “parts” instead of “part” since the method is variadic?

Reply:
Indeed. Fixed in 8b7fbdafdaf956c0c1b6f6f6bf9a90d440c1c339.